### DISCUSSION TITLE
No Feed. No Following. Just my repos.

### DISCUSSION THREAD

**Discussion by slotrans** [community/community#54107] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

First I want to whole-heartedly endorse the sentiment expressed here https://github.com/orgs/community/discussions/53970

Next I want to double down: not only do I not want Github to force the "For You" feed on me, I don't want "Following" either!

When I come to github.com, I should see a list of my repos. That's it. Github is a place to keep code. It is not a social network. Please stop trying to turn it into one. I do not want that. I venture to guess that if you polled Github users, the majority does not want that.

Please, just stop.

**Comment by csilverman** [comment]
100% agreed. I hate this. I don't want a social network. I want my code. If you want to have your little social feed, fine, just show me how to get rid of it so I can get back to the repo view. Put it in another tab or something so I can ignore it and focus on the parts of GitHub that actually matter.

**Comment by slyngshede** [comment]
It also doesn't help that finding your own repos from "frontpage" of github.com is rather hard.
Currently when I go to github.com I see a big blank spot in the middle, because I've disabled everything, and two irrelevant "feeds" on either side. The one on the left makes a little sense, I just don't need it. The one on the right, I have no idea what that's showing me.

I don't really care what other people are doing on Github, or what repos they "star". The page should be for my repositories and perhaps "Hey this pull-request you created requires your attention".
